
`text_config_dict` is provided which will be used to initialize `CLIPTextConfig`. The value `text_config["id2label"]` will be overriden.
/home/qcqced/anaconda3/lib/python3.9/site-packages/transformers/optimization.py:391: FutureWarning: This implementation of AdamW is deprecated and will be removed in a future version. Use the PyTorch implementation torch.optim.AdamW instead, or set `no_deprecation_warning=True` to disable this warning
  warnings.warn(
  0%|                                                                                                                         | 0/4127 [00:00<?, ?it/s]
[1/5] Train & Validation
{'input_ids': tensor([[49406,  9654,  3086,  ..., 49407, 49407, 49407],
        [49406, 34237,   568,  ..., 49407, 49407, 49407],
        [49406,  5352,   539,  ..., 49407, 49407, 49407],
        ...,
        [49406,   320, 21977,  ..., 49407, 49407, 49407],
        [49406,  3865,  1125,  ..., 49407, 49407, 49407],
        [49406, 11352,    85,  ..., 49407, 49407, 49407]], device='cuda:0'), 'attention_mask': tensor([[1, 1, 1,  ..., 0, 0, 0],
        [1, 1, 1,  ..., 0, 0, 0],
        [1, 1, 1,  ..., 0, 0, 0],
        ...,
        [1, 1, 1,  ..., 0, 0, 0],
        [1, 1, 1,  ..., 0, 0, 0],
Traceback (most recent call last):                                                                                            | 0/4127 [00:00<?, ?it/s]
  File "/home/qcqced/anaconda3/lib/python3.9/site-packages/transformers/tokenization_utils_base.py", line 248, in __getattr__
    return self.data[item]
KeyError: 'size'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/qcqced/바탕화면/ML_Test/image2text/train.py", line 49, in <module>
    main('image2text_config.json', CFG)
  File "/home/qcqced/바탕화면/ML_Test/image2text/train.py", line 30, in main
    getattr(train_loop, cfg.loop)(cfg)  # init object
  File "/home/qcqced/바탕화면/ML_Test/image2text/trainer/train_loop.py", line 39, in train_loop
    train_loss, grad_norm, lr = train_input.train_fn(
  File "/home/qcqced/바탕화면/ML_Test/image2text/trainer/trainer.py", line 99, in train_fn
    batch_size = labels.size(0)
  File "/home/qcqced/anaconda3/lib/python3.9/site-packages/transformers/tokenization_utils_base.py", line 250, in __getattr__
    raise AttributeError
AttributeError